docs/CONTRIBUTING.md

```diff
@@ -142,6 +142,69 @@ individual prefix requests, which each get their own discussion thread, pull
 request, and CI/CD runs. Submitters of bulk prefix requests that contain many
 issues may be asked to re-submit as individual prefix requests.
 
+#### Prefix Parking
+
+A prefix and its corresponding semantic space are **substantiated** when it's
+provable that a semantic space exists by one or more of the following:
+
+1. There's a public place where you can get the entire list of terms.
+   Ontologies (e.g., [Gene Ontology](https://bioregistry.io/go)) and databases
+   (e.g., [HGNC](https://bioregistry.io/hgnc)) usually make this pretty
+   straightforwards by offering download links for the ontology or full
+   database.
+2. There's a working, public URI format string that either lets you get HTML,
+   JSON, RDF, or some other kind of information artifact for a given local
+   unique identifier. For example, [OMIM.PS](https://bioregistry.io/omim.ps)
+   doesn't have a way to get a full list of terms but if you have a given local
+   unique identifier, you can use it with the URI format string to retrieve some
+   information about the entity corresponding to that LUI.
+
+A potential less strict third criteria for substantiation could be when
+references to entities in a semantic space (i.e., in the form of CURIEs) can be
+found in public resources or ontologies that are external to the resource in
+which the prefix/semantic space are defined. This is more common for historical
+prefixes (e.g., OpenCyc references appear quite frequently, but this resource
+was taken down more than a decade ago) and is less applicable to new prefix
+requests. Therefore, this relaxed criteria will not be considered as sufficient
+for substantiation.
+
+We define **prefix parking** as a special case of a prefix request in which the
+corresponding resource/semantic space for the prefix does not yet exist or is
+currently under development (and by definition, is not yet substantiated). The
+Bioregistry does not explicitly discourage prefix parking, but new prefix
+requests qualifying as prefix parking require additional guidelines, partially
+motivated by the difficulty of the discussion on
+https://github.com/biopragmatics/bioregistry/issues/359.
+
+1. While it's not typically under the purview of the Bioregistry Review Team to
+   judge the utility of a prefix nor comment on its corresponding design
+   decisions (e.g., choice of local unique identifier scheme, regular expression
+   pattern, URI format string), submitters seeking to park a prefix must both
+   actively publicly seek out and seriously consider suggestions and advice from
+   the Bioregistry Review Team with regards to these matters (e.g., in the issue
+   corresponding to a new prefix request). Submissions unable/unwilling to
+   follow these guidelines will be dismissed and asked to re-submit after their
```

Comment:
@cthoyt - would a caveat here be that the bioregistry review team would be willing to consider "justifications" provided by the submitter that might convince them to bend the best practices?  I know this sounds like it makes the good practices invalid, but we're in the domain of exceptions and we do want people to use bioregistry with preference, but they might not always be able to maneuver into the guidelines in the identifiers publication, etc. I guess what I am asking is for some wiggle room here (provided there is a "good reason" for that wiggle room).

Reply:
I will soften this language - and leave it up to the Bioregistry review team on a case by case basis.

```suggestion
   follow these guidelines may be dismissed and asked to re-submit after their
```